packages/headless/src/features/configuration/configuration-actions.ts

```diff
@@ -34,14 +40,21 @@ export const updateBasicConfiguration = createAction(
  * @param apiBaseUrl (string) The Search API base URL to use (e.g., `https://platform.cloud.coveo.com/rest/search/v2`).
  * @param pipeline (string) The name of the query pipeline to use for the query (e.g., `External Search`). If not specified, the default query pipeline will be used.
  * @param searchHub (string) The first level of origin of the request, typically the identifier of the graphical search interface from which the request originates (e.g., `ExternalSearch`).
+ * @param locale (string) The locale of the current user. Must comply with IETF’s BCP 47 definition: https://www.rfc-editor.org/rfc/bcp/bcp47.txt.
```

Comment:
Could we add an example or maybe some more user friendly documentation, it's hard to actually figure out what a `bcp47` language tag looks like from the specification unless you already know.